gnd/src/main/java/com/google/android/gnd/persistence/sync/TileSourceDownloadWorker.java

```diff
@@ -170,7 +172,7 @@ private Completable processTiles(ImmutableList<Tile> pendingTiles) {
                 case IN_PROGRESS:
                 case FAILED:
                 default:
-                  return downloadTile(t);
+                  return downloadTileSource(t);
```

Comment:
Naming nit: You download tiles from a tile source, so maybe this would make more sense as "downloadTiles"?

Reply:
Fair point. I guess the "Tile > TileSet" renaming issue isn't really a duplication then.

One could argue that we are in fact downloading the tile source--we're simply copying the file specified at a given URL (the tile source). We don't really know anything about individual tiles or care about them until we get down to the tile provider level, which looks at tiles specified in a tile source.

But I also don't want to argue semantics--if it's clearer to use "Tiles" or "TileSet" here I think that's fine--technically speaking though, if we are calling the thing specified in the geoJSON a tile source, that is what we are downloading.

Reply:
Ahhh... sorry, at first glance I may have though this was actually downloading the tiles themselves. Please disregard, this makes perfect sense. If we really wanted to future-proof our semantics, we could define "tile source" to be an arbitrary way of getting tiles, and this case as an "mbtiles tile source's geojson extents file" but we can cross that bridge if we ever get to it.